element_interface/intanloader/rhsutilities.py

```diff
@@ -0,0 +1,1037 @@
+            np.bitwise_and(data["stim_data_raw"], 8192) >= 1
+        )  # get 2^13 bit, interpret as True or False
+        data["stim_polarity"] = 1 - (
+            2 * (np.bitwise_and(data["stim_data_raw"], 256) >> 8)
+        )  # get 2^8 bit, interpret as +1 for 0_bit or -1 for 1_bit
+
+        curr_amp = np.bitwise_and(
+            data["stim_data_raw"], 255
+        )  # get least-significant 8 bits corresponding to the current amplitude
+        data["stim_data"] = (
+            curr_amp * data["stim_polarity"]
+        )  # multiply current amplitude by the correct sign
+
+        # # Scale voltage levels appropriately.
+        data["amplifier_data"] = np.multiply(
+            0.195, (data["amplifier_data"].astype(np.int32) - 32768)
+        )  # units = microvolts
+        data["stim_data"] = np.multiply(
+            header["stim_step_size"], data["stim_data"] / 1.0e-6
+        )
+
+        if header["dc_amplifier_data_saved"]:
+            data["dc_amplifier_data"] = np.multiply(
+                -0.01923, (data["dc_amplifier_data"].astype(np.int32) - 512)
+            )  # units = volts
+
+        data["board_adc_data"] = np.multiply(
+            0.0003125, (data["board_adc_data"].astype(np.int32) - 32768)
+        )  # units = volts
+        data["board_dac_data"] = np.multiply(
+            0.0003125, (data["board_dac_data"].astype(np.int32) - 32768)
+        )  # units = volts
+
+        # Check for gaps in timestamps.
+        num_gaps = np.sum(np.not_equal(data["t"][1:] - data["t"][:-1], 1))
+        if num_gaps == 0:
+            print("No missing timestamps in data.")
+        else:
+            print(
+                "Warning: {0} gaps in timestamp data found.  Time scale will not be uniform!".format(
+                    num_gaps
+                )
+            )
+
+        # Scale time steps (units = seconds).
+        data["t"] = data["t"] / header["sample_rate"]
+
+        # If the software notch filter was selected during the recording, apply the
+        # same notch filter to amplifier data here.
+        if header["notch_filter_frequency"] > 0 and header["version"]["major"] < 3:
+            print_increment = 10
+            percent_done = print_increment
+            for i in range(header["num_amplifier_channels"]):
+                data["amplifier_data"][i, :] = notch_filter(
+                    data["amplifier_data"][i, :],
+                    header["sample_rate"],
+                    header["notch_filter_frequency"],
+                    10,
+                )
+                if fraction_done >= percent_done:
+                    print("{}% done...".format(percent_done))
+                    percent_done = percent_done + print_increment
+    else:
+        data = []
+
+    # Move variables to result struct.
+    result = data_to_result(header, data, data_present)
+
+    print("Done!  Elapsed time: {0:0.1f} seconds".format(time.time() - tic))
+
+    return result, data_present
```

Comment:
```suggestion
    return result
```

I see no point in returning this boolean `data_present`